src/uu/cp/src/copydir.rs

```diff
@@ -99,7 +99,7 @@ impl<'a> Context<'a> {
         let current_dir = env::current_dir()?;
         let root_path = current_dir.join(root);
         let root_parent = if target.exists() && !root.to_str().unwrap().ends_with("/.") {
-            root_path.parent().map(|p| p.to_path_buf())
+            root_path.parent().map(std::path::Path::to_path_buf)
```

Comment:
same comment about Path
```suggestion
            root_path.parent().map(Path::to_path_buf)
```